common/browser_utilities.py

```diff
@@ -12,15 +12,23 @@ def is_text_present_with_retry(text, retries=3, delay=1):
     return False
 
 
-def wait_for(fn, timeout_secs, retry_secs=3.0):
+def wait_for(fn, timeout_secs, retry_secs=3.0, argv=[]):
```

Comment:
argv could and should have better name (as with most things in this function).

Explain how and when argv should be used in the docstring.

Reply:
Also, argv is an empty list here, but above it's passed a string.  The default for this should probably be `None` as the default usually gives a hint as to what a variable should be and this is just confusing.

Reply:
Agreed , argv use is poor. tidying it up to be more explicit